asyncpg_trek/asyncpg.py

```diff
@@ -7,57 +7,59 @@
 from asyncpg_trek._types import Operation
 
 CREATE_TABLE = """\
-CREATE TABLE IF NOT EXISTS migrations (
+CREATE SCHEMA IF NOT EXISTS {schema};
+CREATE TABLE IF NOT EXISTS {schema}.migrations (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS "{schema}.migrations" (
```
Mind adding quotes to these if you have a minute? It avoids issues with weird table / schema names.